examples/nlp/glue_benchmark/glue_benchmark_config.yaml

```diff
@@ -40,14 +40,24 @@ model:
     num_samples: -1
     batch_size: 32
 
+
+  tokenizer:
+      tokenizer_name: bert # or sentencepiece
+      pretrained_model_name: ${model.language_model.pretrained_model_name}
+      vocab_file: null # path to vocab file
+      data_file: null # used to build sentencepiece
+      sample_size: null # for sentencepiece
+      vocab_size: null # vocab size for building sentence piece
+      do_lower_case: null # only used when tokenizer is build with vocab file or with sentencepiece  
+      tokenizer_model: null # only used if tokenizer is sentencepiece
+      special_tokens: null 
+
   language_model:
+    model_type: bert
     pretrained_model_name: bert-base-uncased
     bert_checkpoint: null
-    bert_config: null
-    tokenizer: nemobert # 'nemobert' or 'sentencepiece'
-    vocab_file: null
-    tokenizer_model: null # only used if tokenizer is sentencepiece
-    do_lower_case: false # true for uncased models, false for cased models, will be set automatically if pre-trained tokenizer model is used
+    bert_config_file: null # json file, precedence over bert_config
```

Comment:
Shouldn't it be config_file as it is also for other type of BERT-based models? or they call it bert_config for all?

Reply:
ok, will rename to config_file, It was already used like this the current code in other config files